<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>532</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GPA Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: 57 23pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Grade Point Average</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Enter the number of course</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Enter your last cgpa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: italic 17pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Enter the units for the cgpa</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="no_of_units">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="last_cgpa_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="no_of_courses">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 20pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_result">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(92, 53, 102);
font: 75 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Calculate Result</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
